specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-10-27-preview/privateAccesses.json

```diff
@@ -0,0 +1,475 @@
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/PrivateAccessName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./types/privateAccesses.json#/definitions/PrivateLinkResourceListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all possible private link resources under private access resource.": {
+            "$ref": "./examples/PrivateAccessPrivateLinkResources_Get.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/privateAccesses/{privateAccessName}/privateEndpointConnections/{privateEndpointConnectionName}": {
+      "get": {
+        "tags": [
+          "PrivateAccesses"
+        ],
+        "operationId": "PrivateAccesses_GetAPrivateEndpointConnection",
+        "description": "Gets information about a private endpoint connection under a private access resource.",
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/PrivateAccessName"
+          },
+          {
+            "$ref": "#/parameters/PrivateEndpointConnectionName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/privatelinks.json#/definitions/PrivateEndpointConnection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get information about a private endpoint connection under a private access resource.": {
+            "$ref": "./examples/PrivateAccessPrivateEndpointConnection_Get.json"
+          }
+        }
+      },
+      "delete": {
```

Comment:
>delete [](http://example.com/codeflow?start=7&length=6)

why it dos not have PUT? How will it be created?

Reply:
This PrivateEndpointConnection resource is created by our RP internally. We wait for network RP to setup the private endpoint and then create this connection. User can delete them but not create this themselves. This is pretty standard among all resource that support private endpoint I believe.